tests/lean/macroStack.lean.expected.out

```diff
@@ -21,19 +21,14 @@ but is expected to have type
 failed to synthesize instance
   CoeT String x✝ Nat
 with resulting expansion
-   HasAdd.add._@._internal._hyg.0  ( x
-   +  x  )  x 
+  HasAdd.add✝ (x + x✝) x✝¹
```

Comment:
@leodemoura Pretty printer in action! I did not remove `Syntax.prettyPrint` completely yet since there are two occurrences in pure code...